### DISCUSSION TITLE
Existing Tridactyl Quickmarks

### DISCUSSION THREAD

**Discussion by sdsoderborg** [tridactyl/tridactyl#3324] [Q&A] [answered]
Hi All,

First off Tirydactyl is an awesome tool, I love the functionality and I have not yet scratched the surface of its capability. 

I have been using VIM tools with Firefox for years. One of my favorite features and functions is that of keyboard bindings with the command "quickmark". Vimperator was my goto tool for a long time until it was retired due to a Firefox update, of which I do not recall specifics. As I recall, Tridactyl was the port of the Vimperator functionality. In any case, in Vimperator there was a command sequence to view existing "quickmark" bindings, is there such a function in Tridactyl? Further, do these bindings get written to a flat file that I can view?

Please Advise.

Thanks

**Comment by bovine3dom** [comment, marked as answer]
Currently quickmarks are just a convenient way of making keybinds. The best way to view them is with the rest of your keybinds by, from a normal webpage, using `:viewconfig --user nmaps` and looking at the binds that start with `go`, `gw` and `gn`. The way to remove them is simply `:reset go[the character]` and repeat that for `gw` and `gn`.

We were talking about changing this just a few days ago - https://github.com/tridactyl/tridactyl/issues/3299#issuecomment-770293509 - so we could eventually provide a nicer way for viewing and removing quickmarks.

As for being written to a file: they're stored in the internal Firefox storage which is some sort of database. If you want to write them to a file for version control, you can use `:mkt` if you have `:native` installed, which will make `~/.config/tridactyl/tridactylrc`. If you don't want native installed you can use `:mkt --clipboard` and store the file wherever you want (probably GitHub, because then you can use `:source --url` to read it).

I hope this helps : )

NB: some of these commands might currently only be available in the beta builds ... I always lose track of what's in stable : )
